rx200: #namespace

    # ******** Task Related parameters (we don't use these)
    n_actions: 5  # The number of available actions (default)
    n_actions_joint: 5  # The number of available actions (joint positions)
    n_actions_ee: 3  # The number of available actions (EE position)
    n_observations: 12  # The number of observations
    max_steps: 100  # Maximum number of steps (not currently used)

    # ******** Action Space
    # Define the minimum and maximum joint values
    min_joint_pos:
      - -3.14
      - -1.88
      - -1.88
      - -1.74
      - -3.14

    max_joint_pos:
      - 3.14
      - 1.97
      - 1.62
      - 2.14
      - 3.12

    # ******** Observation Space

    position_ee_max: # Maximum XYZ position for the end effector (EE) (reach is 55cm)
      x: 0.55
      y: 0.55
      z: 0.55
    position_ee_min: # Minimum XYZ position for the EE
      x: -0.55
      y: -0.55
      z: -0.55  # Near zero to prevent going below the table

    rpy_ee_max: # Maximum roll, pitch, and yaw (RPY) values for the EE
      r: 6.2832
      p: 6.2832
      y: 6.2832
    rpy_ee_min: # Minimum rpy position for the EE
      r: -6.2832
      p: -6.2832
      y: -6.2832

    linear_distance_max: # Maximum distance from EE to the desired goal EE (spans 110cm)
      x: 1.00
      y: 1.00
      z: 0.50
    linear_distance_min: # Minimum distance from EE to the desired goal E
      x: -1.00
      y: -1.00
      z: -0.50

    max_distance: 1.00 # Maximum distance from EE to the desired GOAL EE (50cm + 50cm)


    # ******** goal space
    position_goal_max: # Maximum XYZ position for the goal
      x: 0.40
      y: 0.40
      z: 0.40
    position_goal_min: # Minimum XYZ position for the goal
      x: 0.15
      y: -0.40
      z: 0.15


    # ******** achieved_goal and desired_goal

    # In this task this is the EE pose
    position_achieved_goal_max: # Maximum XYZ position for the achieved_goal
      x: 0.55
      y: 0.55
      z: 0.55
    position_achieved_goal_min: # Minimum XYZ position for the achieved_goal
      x: -0.55
      y: -0.55
      z: -0.55

    # In this task this is the same as goal
    position_desired_goal_max: # Maximum XYZ position for the desired_goal
      x: 0.40
      y: 0.40
      z: 0.40
    position_desired_goal_min: # Minimum XYZ position for the desired_goal
      x: 0.15
      y: -0.40
      z: 0.15


    # ******** Gazebo Simulation parameters
    gazebo_time_step: 0.001
    gazebo_update_rate_multiplier: 50.0 # If training multiply the max update rate by this number


    # ******** Tolerance for the position of the goal
    reach_tolerance: 0.02


    # ******** Variables related to rewards
    step_reward: -0.5
    multiplier_dist_reward: 2.0
    reached_goal_reward: 20
    joint_limits_reward: -2.0
    none_exe_reward: -5.0

